### PR TITLE
Join infinispan scenarios in order to reduce native package time

### PR DESCRIPTION
**Pull request by pjgg**
Goal: save native package time.
Desc: Currently we have two Infinispan modules `008-infinispan-getting-started` and `009-quarkus-infinispan-grpc-kafka`.
The first one is a basic scenario that setup an Infinispan app with a single endpoint, secured by a `jks`.Load his credentials from a `hotrod-client.properties`. The second scenario is a more complex app, that load his credencitals from an `application.properties`, and is not secured by a `jks`. 

My proposal is to join both scenarios, loading the credentials from a `hotrod-client.properties` and using a `jks`, with all the test and endpoint of both scenarios.

### CODE REVIEW

**Comment by rsvoboda** [comment]
Adding Kyrylo as he worked on 008 module. 

Maybe the 009 module name could be adjusted to mention `SSL/TLS` addition in https://github.com/quarkus-qe/beefy-scenarios/commit/042a7a9f8c6513ffe489a96bbb469ef7b515afe9

File: 009-quarkus-infinispan-grpc-kafka/src/main/resources/application.properties [hunk 1]
```diff
@@ -1,13 +1,18 @@
...
-quarkus.infinispan-client.auth-server-name=infinispan
-quarkus.infinispan-client.client-intelligence=BASIC
-quarkus.infinispan-client.sasl-mechanism=DIGEST-MD5
+# credentials is picked at build time from hotrod-client.properties
+#quarkus.infinispan-client.auth-username=my_username
+#quarkus.infinispan-client.auth-password=my_password
+#quarkus.infinispan-client.auth-realm=default
+#quarkus.infinispan-client.auth-server-name=infinispan
+#quarkus.infinispan-client.client-intelligence=BASIC
+#quarkus.infinispan-client.sasl-mechanism=DIGEST-MD5
```
Comment (kshpak) [inline 009-quarkus-infinispan-grpc-kafka/src/main/resources/application.properties:15]: I like the comment, but IMO it's redundant to leave props here.
Comment can state that: Infinispan client credentials are picked at build time from hotrod-client.properties

File: 009-quarkus-infinispan-grpc-kafka/src/test/java/io/quarkus/qe/quickstart/InfinispanServerResourceTest.java [hunk 2]
```diff
@@ -1,9 +1,10 @@
-package org.acme.quickstart;
+package io.quarkus.qe.quickstart;
 
 import static io.restassured.RestAssured.given;
 import static org.hamcrest.CoreMatchers.is;
 
-import org.acme.quickstart.containers.InfinispanTestResource;
+
```
Comment (kshpak) [inline 009-quarkus-infinispan-grpc-kafka/src/test/java/io/quarkus/qe/quickstart/InfinispanServerResourceTest.java:6]: Delete extra line

File: 009-quarkus-infinispan-grpc-kafka/src/test/java/io/quarkus/qe/containers/InfinispanTestResource.java [hunk 3]
```diff
@@ -24,7 +25,15 @@
                 .waitingFor(new LogMessageWaitStrategy().withRegEx(".*Infinispan Server.*started in.*\\s"))
                 .withStartupTimeout(Duration.ofMillis(20000))
                 .withEnv("USER", "my_username")
-                .withEnv("PASS", "my_password");
+                .withEnv("PASS", "my_password")
```
Comment (kshpak) [inline 009-quarkus-infinispan-grpc-kafka/src/test/java/io/quarkus/qe/containers/InfinispanTestResource.java:28]: These props are taken from identities.yaml file. Delete these two lines

**Review by kshpak — CHANGES_REQUESTED** [review]
Please also migrate README from the removed module and add a note that it is covered inside quickstart package

**Comment by pjgg** [comment]
> Adding Kyrylo as he worked on 008 module.
> 
> Maybe the 009 module name could be adjusted to mention `SSL/TLS` addition in [042a7a9](https://github.com/quarkus-qe/beefy-scenarios/commit/042a7a9f8c6513ffe489a96bbb469ef7b515afe9)

Added on the Readme

**Comment by rsvoboda** [comment]
rebase pls

**Comment by rsvoboda** [comment]
@kshpak, is the PR OK from your side?

**Comment by kshpak** [comment]
> @kshpak, is the PR OK from your side?

Approve from my side

**Review by kshpak — APPROVED** [review]

**Comment by rsvoboda** [comment]
008 .. there is just readme with pointer to the new location

**Comment by rsvoboda** [comment]
I will wait for CI before approving

**Review by rsvoboda — APPROVED** [review]

**Pull request merged**